test/test_jit.py

```diff
@@ -36,10 +36,21 @@
 import torch.jit._logging
 import torch.jit.frontend
 import torch.jit.quantized
+from torch.jit._recursive import wrap_cpp_module
 import torch.nn as nn
 import torch.nn.functional as F
 from torch.quantization import QConfig
 from torch.quantization._quantize_script import ConvPackedParams, LinearPackedParams
+from torch.quantization._quantize_script import script_qconfig
```

Comment:
We probably need to move all quantization related tests from this file into a separate file (no need to do it in this PR).